integrations/tensorflow-keras/Neptune-TensorFlow-Keras.ipynb

```diff
@@ -0,0 +1,681 @@
+    ]
+   },
+   "source": [
+    "### Step 2: Add NeptuneMonitor Callback to model.fit()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "82879lQuVfsJ",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "from neptunecontrib.monitoring.keras import NeptuneMonitor\n",
+    "\n",
+    "model.fit(x_train, y_train,\n",
+    "          epochs=5,\n",
+    "          batch_size=64,\n",
+    "          callbacks=[NeptuneMonitor()])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 3: Explore results in the Neptune UI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "Go to the experiment link and explore charts that were created from the metric values you logged. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "P04k_nFiXl63",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Step 4: Stop logging"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "When you track experiments with Neptune in Jupyter notebooks you need to explicitly stop the experiment by running ```neptune.stop()```.\n",
+    "\n",
+    "If you are running Neptune in regular ```.py``` scripts it will stop automatically when your code stops running."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "\n",
+    "exp = neptune.get_experiment()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "T5teGbiLwuEi",
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "neptune.stop()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "tests"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# tests\n",
+    "\n",
+    "all_logs = exp.get_logs()\n",
+    "\n",
+    "## check logs\n",
+    "correct_logs = ['batch_loss', 'batch_accuracy', 'epoch_loss', 'epoch_accuracy']\n",
+    "\n",
+    "if set(all_logs.keys()) != set(correct_logs):\n",
+    "    raise ValueError('incorrect metrics')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "YHQrdIgOIo_E",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## More Options"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "You can log way more than just metrics to Neptune. \n",
+    "\n",
+    "Let's go over some of those options here. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Q9zxE-EzLAbe",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Log hardware consumption"
```

Comment:
Maybe add link to install guide or install command also in the comment - user who opens .py in docs will not see instalation cell.